the22mastermind/questioner
# Ft admin update meetup 163420038 endpoint

#### What does this PR do?
Add update meetup functionality with database
#### Description of Task to be completed?
Update of an existing meetup
#### How should this be manually tested?
1. After cloning repo, cd into it and run 'npm run start'
2. Using POSTMAN enter in URL: [PATCH] http://localhost:5000/api/v1/meetups/{meetupID} and you should be prompted for the required fields.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#163420038 https://www.pivotaltracker.com/story/show/163420038
#### Screenshots (if appropriate)
#### Questions:
